ui/src/api/wfe.js

```diff
@@ -40,11 +40,11 @@ router.get('/', async (req, res, next) => {
     const url =
       baseURL2 +
       'search?size=100&sort=startTime:DESC&freeText=' +
-      freeText.join(' AND ') +
+      encodeURIComponent(freeText.join(' AND ')) +
       '&start=' +
       start +
       '&query=' +
-      query;
+      encodeURIComponent(query);
```

Comment:
This should probably also be applied for the `search-by-task` api below.

Reply:
I commited a change to search-by-task.  Place review